src/vmm/src/arch/x86_64/mptable.rs

```diff
@@ -433,12 +433,15 @@ mod tests {
     fn cpu_entry_count_max() {
         let cpus = MAX_SUPPORTED_CPUS + 1;
```

Comment:
I think here we can change the type of the constant to `u8` to avoid casts altogether.